packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -399,74 +387,104 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
       );
 
     const dropdown = isOpened && (
-      <StyledDropdown
+      <ul
+        className={cx(
+          "font-base bg-white-normal lm:absolute lm:inset-x-0 lm:overflow-y-scroll lm:shadow-action lm:rounded-normal z-[3] flex w-full flex-col",
```

Comment:
I think we should enforce `list-none` here, WDYT?

Reply:
Even though it is already present in the tailwind preflight? I was thinking about it but in the end decided to rely just on the preflight to keep the component styles leaner.

Reply:
Ah, my bad, I did not notice it was on preflight. No needed then! Thanks